driver_cpl/cime_config/config_component.xml

```diff
@@ -409,7 +409,7 @@
   <entry id="DOUT_S"> 
     <type>logical</type>
     <valid_values>TRUE,FALSE</valid_values>
-    <default_value>TRUE</default_value>
+    <default_value>FALSE</default_value>
```

Comment:
Currently, we prefer it FALSE. Mark turned DOUT_S to false a few months ago, and seems to prefer it. His reasoning is because he prefers the organization of the log files without this step. I like it because I can run case.run, instead of case.submit (I hope I have it correct). No one has complained that I know of.

In the medium term, we will probably need to have a way of different defaults, since this is likely not the last case.

In the short term, you can pick one, and one team has to deal temporarily, but the next coin-flip goes the other way.